frontend/components/auth/login-page.tsx

```diff
@@ -0,0 +1,86 @@
+"use client";
+
+import Link from "next/link";
+
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { useState } from "react";
+import { useAuth } from "@/app/auth/auth-context";
+import { useRouter } from "next/navigation";
+
+export function LoginForm() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const auth = useAuth();
+  const router = useRouter();
+
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    const user = await auth?.login(email, password);
+    if (user?.isAdmin) {
+      router.push("/admin-user-management");
```

Comment:
Why do we redirect them to "/admin-user-management" page? Is it for the sake of testing that page? Might be better to just redirect to "/" so we don't forget it in the future

Reply:
Thought would be better UX, like how when you login on other websites, you go to the actual content page